[Ready for review] Polish translation Introduction to Web Accessibility (#97)
w3c/wai-intro-accessibility: content/index.pl.md

lang: pl   # Change "en" to the translated-language shortcode
last_updated: 2024-05-31   # Put the date of this translation YYYY-MM-DD (with month in the middle)

# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
translators:
  - name: "Stefan Wajda"
 
contributors:
  - name: "Michał Sobkowiak"   # Replace Jan Doe with contributor name, or delete this line if none

github:
    repository: w3c/wai-intro-accessibility
    branch: gh-pages
    path: content/index.pl.md   # Add the language shortcode to the middle of the filename, for example: content/index.fr.md

permalink: /fundamentals/accessibility-intro/pl   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
ref: /fundamentals/accessibility-intro/     # Do not change this

changelog: /fundamentals/accessibility-intro/changelog/ # Do not change this
layout: default
  
# In the footer below:
# Do not change the dates
# Do not translate CHANGELOG
# Translate the other words, including "Date:" and "Editor:"
# Translate the Working Group name. Leave the Working Group acronym in English.
footer: >

>   <p><strong>Data:</strong> Aktualizacja 20 listopada 2023. Pierwsza publikacja w lutym 2005. DZIENNIK ZMIAN.</p>

Reviewer: @zwiastunsw Please keep "CHANGELOG" untranslated. 

It will automatically create a link to the changelog with the Polish wording from https://github.com/w3c/wai-website-data/blob/master/translations.yml

Author: ```suggestion
  <p><strong>Data:</strong> Aktualizacja 20 listopada 2023. Pierwsza publikacja w lutym 2005. CHANGELOG.</p>
```

Zwiększenie dostępności przynosi korzyści jednostkom, firmom i społeczeństwu.  Międzynarodowe standardy internetowe określają, co jest potrzebne do zapewnienia dostępności.

{::nomarkdown}
{% include box.html type="end" %}
{:/}

{::options toc_levels="2" /}

{::nomarkdown}
{% include_cached toc.html type="start" title="Treść strony" class="full" %}
{:/}

-   TOC is created automatically.
{:toc}

<span class="box-h box-h-simple box-h-full">Materiały pokrewne</span><br>
{% include video-link.html title="Film wprowadzający do dostępności internetowej i standardów W3C <em>(4 minuty)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}

{::nomarkdown}
{% include_cached toc.html type="end" %}
{:/}

## Spojrzenie na dostępność {#context}

<blockquote class="pull">
  <p>Siła Internetu tkwi w jego uniwersalności.<br />

>     Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym atrybutem.</p>

Reviewer: Original quote reads "Access by everone regardless of disability is _**an**_ essential aspect".

It seems the translation reads "is **_its_** essential attribute"

@zwiastunsw Would you adapt the translation?

Author: ```suggestion
    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym aspektem.</p>
```

## Czym jest dostępność internetowa {#what}

Dostępność internetowa oznacza, że ​​strony internetowe, narzędzia i technologie są projektowane i rozwijane w taki sposób, by mogły z nich korzystać osoby z niepełnosprawnościami. Mówiąc dokładniej, ludzie mogą:

-   postrzegać, rozumieć, nawigować i wchodzić w interakcję z Internetem
-   współtworzyć Internet. 

Dostępność internetowa obejmuje wszystkie rodzaje niepełnosprawności wpływające na dostęp do Internetu, w tym niepełnosprawność:

-   słuchu,
-   poznawczą,
-   neurologiczną,
-   fizyczną,
-   mowy,
-   wzroku.

Dostępność Internetu przynosi korzyści także osobom _bez_ niepełnosprawności, na przykład:

-   osobom korzystającym z telefonów komórkowych, inteligentnych zegarków, inteligentnych telewizorów i&nbsp;innych urządzeń z małymi ekranami, różnymi trybami wprowadzania danych itp.
-   osobom starszym ze zmieniającymi się zdolnościami ze względu na starzenie się,
-   osobom z „tymczasową niepełnosprawnością”, np. złamaną ręką lub zgubionymi okularami,

> -   osobom oganiczonym okolicznościami, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,

Reviewer: Original content: "people with “situational limitations”"

@zwiastunsw Could the wording be closer to the original? This terminology has been chosen by the EOWG after [extensive discussion](https://www.w3.org/WAI/EO/wiki/Situational_terminology).

Author: ```suggestion
-   osobom z ograniczeniami sytuacyjnymi, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
```

-   osobom korzystającym z wolnego połączenia internetowego albo mającym ograniczony lub kosztowny dostęp do internetu.

> Zobacz 7-minutowy film z przykładami tego, jak niezbędna jest dostępność dla osób z niepełnosprawnościami i jak przydatna dla wszystkich w rozmaitych sytuacjach:<br>

Reviewer: Original content: "For a 7-minute video with examples of how accessibility is essential for people with disabilities and useful for everyone in a variety of situations, see:"

It seems "essential" is missing in the translation?

Author: This is right on:
- how accessibility is **essential** for people = jak **niezbędna** jest dostępność dla osób
- and **useful** for everyone: i jak **przydatna** dla wszystkich

Reviewer: @zwiastunsw Thanks for the clarification! My tool misled me.

{% include video-link.html title="Oblicza dostępności internetowej <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}

{::nomarkdown}

> {% include box.html type="start" h="3" title="Więcej o dostępności" class="simple aside" %}

Reviewer: Original content: "More info on What is Accessibility"

The translation reads "More about accessibility". Would you think of a closer translation?

Author: ```suggestion
{% include box.html type="start" h="3" title="Więcej informacji o dostępności" class="simple aside" %}
```

Reviewer: @zwiastunsw Sorry for not being specific enough. I was thinking making it clearer it is about "What is" accessibility (when other sections are about how important it is/how to make the Web accessible, etc.).

Author: Ok :) I send PR

{:/}

## Dostępność jest ważna dla osób fizycznych, firm i społeczeństwa {#important}

Internet staje się coraz bardziej istotnym zasobem dla wielu wymiarów życia: edukacji, zatrudnienia, administracji publicznej, biznesu, opieki zdrowotnej, rekreacji i&nbsp;innych. Dostępność Internetu jest niezbędna, aby zapewnić równy dostęp i&nbsp;równe szanse osobom o&nbsp;różnorodnych możliwościach. Dostęp do informacji i komunikacji międzyludzkiej, w tym Internetu, zdefiniowana jest jako podstawowe prawo człowieka w Konwencji Narodów Zjednoczonych o Prawach osób z niepełnosprawnościami (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).

Internet oferuje możliwość bezprecedensowego dostępu do informacji i&nbsp;interakcji wielu osobom z&nbsp;niepełnosprawnościami. Oznacza to, że bariery w dostępie do mediów drukowanych, dźwiękowych i&nbsp;wizualnych można znacznie łatwiej pokonać za pomocą technologii internetowych.

Dostępność wspiera włączenie społeczne osób z niepełnosprawnościami, a także innych osób, na&nbsp;przykład:

-   ludzi starszych
-   ludzi na obszarach wiejskich
-   ludzi w krajach rozwijających się.

> Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.

Reviewer: A paragraph is missing in the translation :

> There is also a strong business case for accessibility. As shown in the previous section, accessible design improves overall user experience and satisfaction, especially in a variety of situations, across different devices, and for older users. Accessibility can enhance your brand, drive innovation, and extend your market reach.

Author: ```suggestion
Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.

**Dostępność ma również silne uzasadnienie biznesowe.** Jak pokazano w poprzedniej sekcji, dostępny projekt poprawia ogólne doświadczenie i satysfakcję użytkownika, szczególnie w różnych sytuacjach, na różnych urządzeniach i w przypadku starszych użytkowników. Dostępność może wzmocnić markę, pobudzić innowacje i rozszerzyć zasięg rynkowy.
```

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej informacji o znaczeniu dostępności" class="simple aside" %}
{:/}

-   Ogólne informacje o korzyściach dla przedsiębiorstw i przedsięwzięć znajdują się w dokumencie [[Biznesowe uzasadnienie dostępności cyfrowej]](/business-case/).
    -   Przykłady korzyści płynących z [tworzenia dostępnych mediów audio i&nbsp;wideo](/media/av/) przedstawiono w&nbsp;części [Korzyści dla organizacji](/media/av/users-orgs/#benefits).
-   Wskazówki dla ustalania wymogów prawnych znajdują się w&nbsp;zarchiwizowanym opracowaniu [Czynniki prawne i&nbsp;polityczne](https://www.w3.org/WAI/business-case/archive/pol).

{::nomarkdown}
{% include box.html type="end" %}
{:/}

## Tworzyć dostępny Internet {#making}

Dostępność Internetu zależy od kilku współpracujących ze sobą komponentów, w tym technologii internetowych, przeglądarek internetowych i innych „programów użytkownika”, narzędzi do tworzenia treści i&nbsp;witryn internetowych.

> Inicjatywa na Rzecz Dostępności Internetu W3C ([WAI](/about/participating/)) opracowuje specyfikacje techniczne, wytyczne, techniki i&nbsp;materiały pomocnicze opisujące rozwiązania w zakresie dostępności. Są one uznawane za międzynarodowe standardy dostępności internetowej; na przykład <abbr title="Wytyczne dla dostępności treści internetowej (WCAG)">WCAG 2.0 są także normą <abbr title="International Organization for Standardization">ISO</abbr>: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.

Reviewer: @zwiastunsw The first `<abbr>` is missing a closing tag.

Author: ```suggestion
Inicjatywa na Rzecz Dostępności Internetu W3C ([WAI](/about/participating/)) opracowuje specyfikacje techniczne, wytyczne, techniki i&nbsp;materiały pomocnicze opisujące rozwiązania w zakresie dostępności. Są one uznawane za międzynarodowe standardy dostępności internetowej; na przykład <abbr title="Wytyczne dla dostępności treści internetowej (WCAG)">WCAG</abbr> 2.0 są także normą <abbr title="International Organization for Standardization">ISO</abbr>: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.
```

github:
    repository: w3c/wai-intro-accessibility
    branch: gh-pages
    path: content/index.pl.md   # Add the language shortcode to the middle of the filename, for example: content/index.fr.md

permalink: /fundamentals/accessibility-intro/pl   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
ref: /fundamentals/accessibility-intro/     # Do not change this

changelog: /fundamentals/accessibility-intro/changelog/ # Do not change this
layout: default
  
# In the footer below:
# Do not change the dates
# Do not translate CHANGELOG
# Translate the other words, including "Date:" and "Editor:"
# Translate the Working Group name. Leave the Working Group acronym in English.
footer: >
  <p><strong>Data:</strong> Aktualizacja 20 listopada 2023. Pierwsza publikacja w lutym 2005. DZIENNIK ZMIAN.</p>
  <p><strong>Redaktorka:</strong> <a href="http://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
  <p>Opracowane przez Grupę Roboczą ds. Edukacji i Promocji (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>).</p>
---

{::nomarkdown}
{% include box.html type="start" h="2" title="Podsumowanie" class="full" %}
{:/}

> Gdy strony i narzędzia internetowe są odpowiednio zaprojektowane i zakodowane, osoby z niepełnosprawnościami mogą z&nbsp;nich korzystać. Jednak obecnie wiele witryn i narzędzi jest tworzonych z barierami dostępności, które utrudniają lub uniemożliwiają korzystanie z nich niektórym osobom.

Reviewer 2: ```suggestion
Gdy strony i narzędzia internetowe są odpowiednio zaprojektowane i napisane, mogą z nich korzystać osoby z niepełnosprawnościami. Obecnie jednak wiele witryn i narzędzi jest tworzonych z barierami dostępności, które utrudniają lub uniemożliwiają korzystanie z nich niektórym osobom.
```

> Zapewnienie dostępności internetowej przynosi korzyści osobom fizycznym, firmom i&nbsp;społeczeństwu. Międzynarodowe standardy internetowe określają, co jest potrzebne do zapewnienia dostępności.

Reviewer 2: ```suggestion
Zwiększenie dostępności przynosi korzyści jednostkom, firmom i społeczeństwu.  Międzynarodowe standardy internetowe określają, co jest potrzebne do zapewnienia dostępności.
```

{::nomarkdown}
{% include box.html type="end" %}
{:/}

{::options toc_levels="2" /}

{::nomarkdown}
{% include_cached toc.html type="start" title="Treść strony" class="full" %}
{:/}

-   TOC is created automatically.
{:toc}

<span class="box-h box-h-simple box-h-full">Materiały pokrewne</span><br>
{% include video-link.html title="Film wprowadzający do dostępności internetowej i standardów W3C (4 minuty) <em>(4 minuty)</em>" href="https://www.w3.org/WAI/videos/standards-and-benefits/" src="/content-images/wai-intro-accessibility/video-still-accessibility-intro-16-9.jpg" %}

{::nomarkdown}
{% include_cached toc.html type="end" %}
{:/}

> ## Dostępność w kontekście {#context}

Reviewer 2: ```suggestion
##Spojrzenie na dostępność {#context}
```

<blockquote class="pull">
  <p>Siła Internetu tkwi w jego uniwersalności.<br />

>     Dostęp dla wszystkich bez względu na niepełnosprawność jest jego istotnym aspektem.</p>

Reviewer 2: ```suggestion
    Dostęp dla każdego bez względu na niepełnosprawność jest jego podstawowym atrybutem.</p>
```
lub
```suggestion
    Dostęp dla każdego bez względu na niepełnosprawność jest jego zasadniczym atrybutem.</p>
```

  <footer><cite>Tim Berners-Lee, Dyrektor W3C i twórca World Wide Web</cite></footer>
</blockquote>

> Internet został zaprojektowany zasadniczo tak, aby służył wszystkim ludziom, niezależnie od ich sprzętu, oprogramowania, języka, lokalizacji czy sprawności. Gdy Internet osiąga ten cel, jest dostępny dla osób o zróżnicowanym zakresie słuchu, ruchu, wzroku i zdolności poznawczych.

Reviewer 2: ```suggestion
Internet został zasadniczo zaprojektowany tak, aby służył wszystkim, niezależnie od ich urządzeń, oprogramowania, języka, lokalizacji czy sprawności. Gdy Internet osiąga ten cel, jest dostępny dla osób z różnorodnym zakresem zdolności słuchowych, ruchowych, wzrokowych i poznawczych.
```

> W ten sposób wpływ niepełnosprawności ulega radykalnej zmianie w Internecie, bo Internet usuwa bariery w komunikacji i&nbsp;interakcjach, z którymi wiele osób boryka się w świecie fizycznym. Ale źle zaprojektowane strony internetowe, aplikacje, technologie lub narzędzia mogą stworzyć bariery uniemożliwiające ludziom korzystanie z Internetu.

Reviewer 2: ```suggestion
Dlatego wpływ niepełnosprawności zmienia się radykalnie w Internecie, bowiem Internet usuwa bariery w komunikacji i uczestnictwie, z którymi wielu ludzi mierzy się w świecie fizycznym. Z drugiej strony, gdy strony internetowe, aplikacje, technologie lub narzędzia są źle zaprojektowane, mogą stawiać bariery uniemożliwiające ludziom korzystanie z Internetu.
```

> **Dostępność jest niezbędna dla programistów i organizacji, które chcą tworzyć wysokiej jakości strony i&nbsp;narzędzia internetowe, a&nbsp;nie wykluczać ludzi z&nbsp;korzystania z&nbsp;ich produktów i&nbsp;usług.**

Reviewer 2: ```suggestion
**Dostępność jest niezastąpiona dla programistów i organizacji, które chcą tworzyć wysokiej jakości strony i narzędzia internetowe, zamiast wykluczać ludzi z korzystania ze swoich produktów i usług.**
```

Reviewer 2: ew. grupy ludzi

> Siła Internetu tkwi w jego uniwersalności.  
> Dostęp dla wszystkich bez względu na niepełnosprawność jest jego istotnym aspektem.
> 
> Tim Berners-Lee, Dyrektor W3C i twórca World Wide Web

> ## Co to jest dostępność internetowa {#what}

Reviewer 2: ```suggestion
##Czym jest dostępność internetowa {#what}
```

> Dostępność internetowa oznacza, że ​​strony internetowe, narzędzia i technologie są projektowane i rozwijane w taki sposób, aby mogły z nich korzystać osoby z niepełnosprawnościami. Mówiąc dokładniej, ludzie mogą:

Reviewer 2: ```suggestion
Dostępność internetowa oznacza, że ​​strony internetowe, narzędzia i technologie są projektowane i rozwijane w taki sposób, by mogły z nich korzystać osoby z niepełnosprawnościami. Mówiąc dokładniej, ludzie mogą:
```

> -   postrzegać, rozumieć, nawigować i wchodzić w interakcję z Internetem

Reviewer 2: ```suggestion
-   odczuwać, rozumieć, nawigować oraz uczestniczyć w Internecie
```

Author: > 
```suggestion
-   postrzegać, rozumieć, nawigować i wchodzić w interakcję z Internetem
```
postrzegać - słowo  ma kluczowe znaczenie, związana jest z nim 1. zasada - postrzegalności (treści muszą być postrzegalne)
wchodzić w interakcję - tu nie chodzi tylko o uczestnictwo (uczestnictwo może miec bierny charakter)

-   współtworzyć Internet. 

> Dostępność internetowa dotyczy wszystkich rodzajów niepełnosprawności wpływających na dostęp do Internetu, w tym:

Reviewer 2: ```suggestion
Dostępność internetowa obejmuje wszystkie rodzaje niepełnosprawności wpływające na dostęp do Internetu, w tym niepełnosprawność:
```

> -   słuchu

Reviewer 2: ```suggestion
-   słuchu,
```

> -   poznawczej

Reviewer 2: ```suggestion
-   poznawczą,
```

> -   neurologicznej

Reviewer 2: ```suggestion
-   neurologiczną,
```

> -   fizycznej

Reviewer 2: ```suggestion
-   fizyczną,
```

> -   mowy

Reviewer 2: ```suggestion
-   mowy,
```

-   wzroku.

Dostępność Internetu przynosi korzyści także osobom _bez_ niepełnosprawności, na przykład:

-   osobom korzystającym z telefonów komórkowych, inteligentnych zegarków, inteligentnych telewizorów i&nbsp;innych urządzeń z małymi ekranami, różnymi trybami wprowadzania danych itp.

> -   osobom starszym ze zmieniającymi się zdolnościami w związku ze starzeniem się

Reviewer 2: ```suggestion
-   osobom starszym ze zmieniającymi się zdolnościami ze względu na starzenie się,
```

> -   osobom z „tymczasową niepełnosprawnością”, np. złamaną ręką lub zgubionymi okularami

Reviewer 2: ```suggestion
-   osobom z „tymczasową niepełnosprawnością”, np. złamaną ręką lub zgubionymi okularami,
```

> -   osobom z „ograniczeniami sytuacyjnymi”, np. w jasnym świetle słonecznym lub w&nbsp;środowisku, w&nbsp;którym nie mogą słuchać dźwięku

Reviewer 2: ```suggestion
-   osobom oganiczonym okolicznościami, np. jasnym światłem słonecznym lub w otoczeniu, w którym nie mogą słuchać dźwięku,
```

> -   osobom korzystającym z wolnego połączenia internetowego lub mającym ograniczoną albo kosztowną przepustowość.

Reviewer 2: the translation of "bandwidth" bothers me a lot, maybe:
```suggestion
-   osobom korzystającym z wolnego połączenia internetowego albo mającym ograniczony lub kosztowny dostęp do internetu.
```

> -   osobom korzystającym z wolnego połączenia internetowego lub mającym ograniczoną albo kosztowną przepustowość.

Reviewer 2: albo:
```suggestion
-   osobom korzystającym z wolnego połączenia internetowego albo mającym do niego ograniczony lub drogi dostęp.
```

> Zobacz 7-minutowy film z przykładami tego, jak dostępność jest niezbędna dla osób z niepełnosprawnościami i przydatna dla wszystkich w różnych sytuacjach:<br>

Reviewer 2: ```suggestion
Zobacz 7-minutowy film z przykładami tego, jak niezbędna jest dostępność dla osób z niepełnosprawnościami i jak przydatna dla wszystkich w rozmaitych sytuacjach:<br>
```

> {% include video-link.html title="Perspektywy dostępności internetowej <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}

Reviewer 2: ```suggestion
{% include video-link.html title="Oblicza dostępności internetowej <em>(YouTube)</em>" href="https://www.youtube.com/watch?v=3f31oufqFSM" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}
```

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej o dostępności" class="simple aside" %}
{:/}

> -   Jeśli chcesz dowiedzieć się więcej o tym, jak różne rodzaje niepełnosprawności 

Reviewer 2: ```suggestion
-   Jeśli chcesz dowiedzieć się więcej o tym, jaki wpływ na korzystanie z Internetu mają różne rodzaje niepełnosprawności, lub przeczytać o scenariuszach korzystania z Internetu przez osoby z niepełnosprawnościami, zobacz [Jak osoby z niepełnosprawnościami korzystają z Internetu](/people-use-web/).
```

>     wpływają na korzystanie z Internetu i&nbsp;przeczytać o scenariuszach korzystania 

Reviewer 2: ```suggestion
```

> 	z&nbsp;Internetu przez osoby z&nbsp;niepełnosprawnościami, zobacz

Reviewer 2: ```suggestion
```

> 	[Jak osoby z niepełnosprawnościami korzystają z&nbsp;Internetu](/people-use-web/).

Reviewer 2: ```suggestion
```

> -   Jeśli chcesz poznać więcej przykładów korzyści dla innych, 

Reviewer 2: ```suggestion
-   Jeśli chcesz poznać więcej przykładów korzyści dla innych, zapoznaj się z materiałami multimedialnymi:
```

>     zapoznaj się z materiałami multimedialnymi 

Reviewer 2: ```suggestion
```

>     [Używane przez osoby z niepełnosprawnością i bez niej](/media/av/users-orgs/#situations), 

Reviewer 2: ```suggestion
    [Użytkowanie internetu przez osoby z niepełnosprawnością i bez niej](/media/av/users-orgs/#situations), 
```

> 	zarchiwizowanym materiałem [Dostępność internetowa jest korzystna dla osób z&nbsp;niepełnosprawnością i&nbsp;bez&nbsp;niej](https://www.w3.org/WAI/business-case/archive/soc#groups) 

Reviewer 2: ```suggestion
	[Walory dostępności internetowej dla osób z niepełnosprawnością oraz bez niej](https://www.w3.org/WAI/business-case/archive/soc#groups) 
	(materiał archiwalny)
```

> 	oraz zarchiwizowanym zasobem mobilnym [Wspólne doświadczenia internetowe: 

Reviewer 2: ```suggestion
	oraz archiwalnym materiałem [Wspólne doświadczenia internetowe: 
```

> 	Typowe bariery dla 	użytkowników urządzeń mobilnych i&nbsp;osób z&nbsp;niepełnosprawnościami](/standards-guidelines/shared-experiences/).

Reviewer 2: ```suggestion
	Bariery wspólne dla użytkowników urządzeń mobilnych i osób z niepełnosprawnościami](/standards-guidelines/shared-experiences/).
```

{::nomarkdown}
{% include box.html type="end" %}
{:/}

> ## Dostępność jest ważna dla osób fizycznych, firm i społeczeństwa {#important}

Reviewer 2: ```suggestion
##Dostępność jest ważna dla człowieka, firm i społeczeństwa{#important}
```

Author: ```suggestion
## Dostępność jest ważna dla jednostek, firm i społeczeństwa {#important}
```
Tak będzie zgodnie z wcześniejszym zapisem

> Internet staje się coraz bardziej istotnym zasobem dla wielu aspektów życia: edukacji, zatrudnienia, administracji publicznej, biznesu, opieki zdrowotnej, rekreacji i&nbsp;innych. Dostępność Internetu jest niezbędna, aby zapewnić równy dostęp i&nbsp;równe szanse osobom o&nbsp;różnorodnych sprawnościach. Dostępność technologii informacyjnych i&nbsp;komunikacyjnych, w&nbsp;tym Internetu, zdefiniowana jest jako podstawowe prawo człowieka w Konwencji Narodów Zjednoczonych o Prawach osób z&nbsp;niepełnosprawnościami (UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).

Reviewer 2: ```suggestion
Internet staje się coraz bardziej istotnym zasobem dla wielu wymiarów życia: edukacji, zatrudnienia, administracji publicznej, biznesu, opieki zdrowotnej, rekreacji i&nbsp;innych. Dostępność Internetu jest niezbędna, aby zapewnić równy dostęp i&nbsp;równe szanse osobom o&nbsp;różnorodnych możliwościach. Dostęp do informacji i komunikacji międzyludzkiej, w tym Internetu, zdefiniowana jest jako podstawowe prawo człowieka w Konwencji Narodów Zjednoczonych o Prawach osób z niepełnosprawnościami(UN [CRPD](https://www.un.org/development/desa/disabilities/convention-on-the-rights-of-persons-with-disabilities.html)).
```
Za polskim tłumaczeniem dokumentu: https://www.unic.un.org.pl/dokumenty/Konwencja_Praw_Osob_Niepelnosprawnych.pdf
![image](https://github.com/w3c/wai-intro-accessibility/assets/2302121/502f0f6c-4526-4a9d-9b71-14ce5faea347)

Internet oferuje możliwość bezprecedensowego dostępu do informacji i&nbsp;interakcji wielu osobom z&nbsp;niepełnosprawnościami. Oznacza to, że bariery w dostępie do mediów drukowanych, dźwiękowych i&nbsp;wizualnych można znacznie łatwiej pokonać za pomocą technologii internetowych.

Dostępność wspiera włączenie społeczne osób z niepełnosprawnościami, a także innych osób, na&nbsp;przykład:

-   ludzi starszych
-   ludzi na obszarach wiejskich
-   ludzi w krajach rozwijających się.

Dostępność internetowa jest w&nbsp;wielu sytuacjach **wymagana przez prawo**.

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej informacji o znaczeniu dostępności" class="simple aside" %}
{:/}

> -   Ogólne informacje o korzyściach biznesowych znajdują się w dokumencie [[Biznesowe uzasadnienie dostępności cyfrowej]](/business-case/).

Reviewer 2: ```suggestion
-   Ogólne informacje o korzyściach dla przedsiębiorstw i przedsięwzięć znajdują się w dokumencie [[Biznesowe uzasadnienie dostępności cyfrowej]](/business-case/).
```

    -   Przykłady korzyści płynących z [tworzenia dostępnych mediów audio i&nbsp;wideo](/media/av/) przedstawiono w&nbsp;części [Korzyści dla organizacji](/media/av/users-orgs/#benefits).
-   Wskazówki dla ustalania wymogów prawnych znajdują się w&nbsp;zarchiwizowanym opracowaniu [Czynniki prawne i&nbsp;polityczne](https://www.w3.org/WAI/business-case/archive/pol).

{::nomarkdown}
{% include box.html type="end" %}
{:/}

## Tworzyć dostępny Internet {#making}

> Dostępność Internetu zależy od kilku współpracujących ze sobą komponentów, w tym technologii internetowych, przeglądarek internetowych i innych „programów użytkownika”, narzędzi autorskich i&nbsp;witryn internetowych.

Reviewer 2: ```suggestion
Dostępność Internetu zależy od kilku współpracujących ze sobą komponentów, w tym technologii internetowych, przeglądarek internetowych i innych „programów użytkownika”, narzędzi do tworzenia treści i&nbsp;witryn internetowych.
```

Inicjatywa na Rzecz Dostępności Internetu W3C ([WAI](/about/participating/)) opracowuje specyfikacje techniczne, wytyczne, techniki i&nbsp;materiały pomocnicze opisujące rozwiązania w zakresie dostępności. Są one uznawane za międzynarodowe standardy dostępności internetowej; na przykład <abbr title="Wytyczne dla dostępności treści internetowej (WCAG)">WCAG 2.0 są także normą <abbr title="International Organization for Standardization">ISO</abbr>: ISO/<abbr title="International Electrotechnical Commission">IEC</abbr> 40500.

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej o tworzeniu dostępnego Internetu" class="simple aside" %}
{:/}

-   Więcej informacji o tych aspektach współdziałania w zakresie dostępności można znaleźć w&nbsp;artykule [[Podstawowe komponenty dostępności internetowej]](/fundamentals/components/).

> -   Wytyczne dla dostępności treści internetowych (WCAG), Wytyczne dla dostępności narzędzi autorskich (ATAG), ARIA dla złożonych dostępnych aplikacji internetowych i inne ważne materiały zostały przedstawione w&nbsp;[[Omówieniu standardów dostępności W3C]](/standards-guidelines/).

Reviewer 2: ```suggestion
-   Wytyczne dla dostępności treści internetowych (WCAG), Wytyczne dla dostępności narzędzi do tworzenia treści (ATAG), ARIA dla złożonych dostępnych aplikacji internetowych i inne ważne materiały zostały przedstawione w&nbsp;[[Omówieniu standardów dostępności W3C]](/standards-guidelines/).
```

> -   Aby dowiedzieć się więcej o tym, jak W3C WAI opracowuje materiały poprzez wielostronne, międzynarodowe uczestnictwo i&nbsp;jak możesz wnieść swój wkład, zobacz [O WAI](/about/) oraz [Uczestnictwo w&nbsp;WAI](/about/participating/).

Reviewer 2: ```suggestion
-   Aby dowiedzieć się więcej o tym, jak W3C WAI opracowuje materiały poprzez wielostronne, międzynarodowe uczestnictwo i&nbsp;jak możesz pomóc, zobacz [O WAI](/about/) oraz [Uczestnictwo w&nbsp;WAI](/about/participating/).
```

{::nomarkdown}
{% include box.html type="end" %}
{:/}

> ### Uczyń dostępną swoją witrynę {#website}

Reviewer 2: ```suggestion
###Spraw by Twoja witryna była dostępna{#website}
```

> Wiele aspektów dostępności można łatwo zrozumieć i wdrożyć. Część rozwiązań z&nbsp;zakresu dostępności jest bardziej złożona i&nbsp;wymaga większej wiedzy do wdrożenia.

Reviewer 2: ```suggestion
Wiele aspektów dostępności można stosunkowo łatwo zrozumieć i wdrożyć. Część rozwiązań z&nbsp;zakresu dostępności jest jednak bardziej złożona i&nbsp; wdrożenie ich wymaga większej wiedzy.
```

> Najbardziej wydajne i skuteczne jest uwzględnianie dostępności od samego początku projektu, dzięki czemu nie trzeba wracać i&nbsp;ponownie wykonywać pracy.

Reviewer 2: ```suggestion
Najbardziej wydajne i skuteczne jest uwzględnianie dostępności od samego początku projektu, dzięki czemu nie trzeba ponownie wykonywać tej samej pracy.
```

-   Gdy tylko poczujesz się na siłach, by dowiedzieć się więcej o&nbsp;programowaniu i&nbsp;projektowaniu, prawdopodobnie skorzystasz z&nbsp;takich materiałów jak:
    -   [Jak spełnić WCAG (Krótki przewodnik)](https://www.w3.org/WAI/WCAG22/quickref/)
    -   [Samouczki dotyczące dostępności internetowej](/tutorials/)
-   Aby poznać problemy zarządzania projektami i organizacją, zobacz[Planowanie i zarządzanie dostępnością internetową](/planning-and-managing/).<br>
    Jeśli chcesz już teraz dokonać szybkich poprawek, zobacz temat [Sposoby napraw tymczasowych](/planning/interim-repairs/).

{::nomarkdown}
{% include box.html type="end" %}
{:/}
	
## Ocena dostępności {#evaluate}

Podczas tworzenia lub modernizacji strony internetowej, należy oceniać dostępność na wczesnym etapie i&nbsp;w trakcie całego procesu tworzenia, aby rozpoznać problemy z&nbsp;dostępnością jak najwcześniej, kiedy łatwiej jest im zaradzić. Proste kroki, jak na przykład zmiana ustawień przeglądarki, mogą pomóc w&nbsp;ocenie niektórych aspektów dostępności. Kompleksowa ocena w&nbsp;celu ustalenia, czy witryna spełnia wszystkie wytyczne dotyczące dostępności, wymaga więcej wysiłku.

> Istnieją specjalne narzędzia, które pomagają w ocenie. Ale żadne narzędzie nie potrafi samodzielnie określić, czy witryna spełnia wytyczne dotyczące dostępności. Aby określić, czy witryna jest dostępna, niezbędna jest kompetentna ocena człowieka.

Reviewer 2: ```suggestion
Istnieją specjalne narzędzia, które pomagają w ocenie. Jednak żadne narzędzie nie potrafi samodzielnie określić, czy witryna spełnia wytyczne dotyczące dostępności. Aby określić, czy witryna jest dostępna, niezbędna jest kompetentna ocena człowieka.
```

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej o ocenianiu dostępności" class="simple aside" %}
{:/}

-   Materiały pomocne w ocenie dostępności przedstawiono w&nbsp;sekcji [[Ocena dostępności internetowej]](/test-evaluate/).

{::nomarkdown}
{% include box.html type="end" %}
{:/}

{% include excol.html type="start" id="examples" %}

## Przykłady

{% include excol.html type="middle" %}

### Tekst alternatywny dla obrazów

![obraz logo; Znacznik HTML img alt='Logo Inicjatywy na Rzecz Dostępności Internetowej'](https://www.w3.org/WAI/intro/alt-logo.png){:.right}

> Obrazy powinny mieć *[równoważną alternatywę tekstową](https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives)* (tekst alt) w znacznikach/kodzie.

Reviewer 2: ```suggestion
Obrazy powinny mieć *[równznaczną alternatywę tekstową](https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives)* (tekst alt) w znacznikach/kodzie.
```

Author: ```suggestion
Obrazy powinny mieć *[równoznaczną alternatywę tekstową](https://www.w3.org/WAI/WCAG22/Understanding/text-alternatives)* (tekst alt) w znacznikach/kodzie.
```
literówka

Jeśli nie istnieje tekst alternatywny dla obrazów, informacje o&nbsp;obrazie są niedostępne, na przykład dla osób, które nie widzą i&nbsp;używają czytnika ekranu, który odczytuje na głos informacje na stronie, w&nbsp;tym tekst alternatywny dla obrazów graficznych.

> Gdy podany jest równoważny tekst alternatywny, informacje są dostępne dla osób niewidomych, a&nbsp;także dla osób, które wyłączają obrazy (na przykład w regionach o&nbsp;drogiej lub niskiej przepustowości). Są one również dostępne dla technologii, które nie widzą obrazów, takich jak wyszukiwarki.

Reviewer 2: ```suggestion
Gdy podany jest równznaczny tekst alternatywny, informacje są dostępne dla osób niewidomych, a&nbsp;także dla osób, które wyłączają obrazy (na przykład w regionach o&nbsp;drogiej lub niskiej przepustowości sieci). Są one również dostępne dla technologii, które nie widzą obrazów, takich jak wyszukiwarki.
```

Author: ```suggestion
Gdy podany jest równoznaczny tekst alternatywny, informacje są dostępne dla osób niewidomych, a&nbsp;także dla osób, które wyłączają obrazy (na przykład w regionach o&nbsp;drogiej lub niskiej przepustowości sieci). Są one również dostępne dla technologii, które nie widzą obrazów, takich jak wyszukiwarki.
```
Literówka

### Wprowadzanie danych z klawiatury

![przekreślona mysz](https://www.w3.org/WAI/intro/no-mouse.png){:.left width="67" height="45"}

> Niektóre osoby nie mogą używać myszy, między innymi wielu starszych użytkowników z&nbsp;ograniczoną precyzyjną kontrolą motoryczną. Dostępna witryna internetowa nie opiera się na myszy; sprawia, że [cała funkcjonalność jest dostępna z&nbsp;poziomu klawiatury](https://www.w3.org/WAI/WCAG22/Understanding/keyboard-accessible). Dzięki temu osoby z&nbsp;niepełnosprawnościami będą mogły korzystać z&nbsp;[technologii wspomagających](/planning/involving-users/#at), które symulują klawiaturę, takie jak wprowadzanie mowy.

Reviewer 2: ```suggestion
Niektóre osoby nie mogą używać myszy, między innymi wielu starszych użytkowników z&nbsp;ograniczoną precyzyjną kontrolą motoryczną. Dostępna witryna internetowa nie opiera się na myszy; sprawia, że [cała funkcjonalność jest dostępna z&nbsp;poziomu klawiatury](https://www.w3.org/WAI/WCAG22/Understanding/keyboard-accessible). Dzięki temu osoby z&nbsp;niepełnosprawnościami mogą korzystać z&nbsp;[technologii wspomagających](/planning/involving-users/#at), zastępujących klawiaturę, takich jak sterowanie głosowe.
```

### Transkrypcje dźwięku

[![przykładowa transkrypcja](https://www.w3.org/WAI/intro/transcript.png){:.right width="251" height="254"}](http://www.w3.org/WAI/highlights/200606wcag2interview.html)

> Tak jak obrazy nie są dostępne dla osób, które nie widzą, tak pliki audio nie są dostępne dla osób, które nie słyszą. Zapewnienie transkrypcji tekstowej sprawia, że ​​informacje audio są dostępne dla osób niesłyszących lub niedosłyszących, a&nbsp;także dla wyszukiwarek i&nbsp;innych technologii, które nie słyszą.

Reviewer 2: ```suggestion
Tak jak obrazy nie są dostępne dla osób, które nie widzą, tak pliki audio nie są dostępne dla osób, które nie słyszą. Zapewnienie transkrypcji sprawia, że ​​informacje audio są dostępne dla osób niesłyszących lub niedosłyszących, a&nbsp;także dla wyszukiwarek i&nbsp;innych technologii, które nie słyszą.
```

> Tak jak obrazy nie są dostępne dla osób, które nie widzą, tak pliki audio nie są dostępne dla osób, które nie słyszą. Zapewnienie transkrypcji tekstowej sprawia, że ​​informacje audio są dostępne dla osób niesłyszących lub niedosłyszących, a&nbsp;także dla wyszukiwarek i&nbsp;innych technologii, które nie słyszą.

Reviewer 2: ewentualnie:
```suggestion
Tak jak obrazy nie są dostępne dla osób, które nie widzą, tak pliki audio nie są dostępne dla osób, które nie słyszą. Zapewnienie tekstu transkrypcji sprawia, że ​​informacje audio są dostępne dla osób niesłyszących lub niedosłyszących, a&nbsp;także dla wyszukiwarek i&nbsp;innych technologii, które nie słyszą.
```

Zapewnienie transkrypcji w witrynach internetowych jest łatwe i&nbsp;stosunkowo niedrogie. Istnieją również [usługi transkrypcji](http://www.uiaccess.com/transcripts/transcript_services.html), które tworzą transkrypcje tekstu w&nbsp;formacie HTML.

{::nomarkdown}
{% include box.html type="start" h="3" title="Więcej przykładów" class="simple aside" %}
{:/} 

-   [Wskazówki na dobry początek](/tips/)
-   [[Łatwe testy - Wstępna ocena dostępności]](/test-evaluate/preliminary/)

> -   {% include video-link.html class="small inline" title="Perspektywy dostępności internetowej &mdash; filmy i opisy" href="/perspective-videos/" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}

Reviewer 2: ```suggestion
-   {% include video-link.html class="small inline" title="Oblicza dostępności internetowej &mdash; filmy i opisy" href="/perspective-videos/" src="/content-images/wai-intro-accessibility/video-still-accessibility-perspectives-16-9.jpg" %}
```

{::nomarkdown}
{% include box.html type="end" %}
{:/}

{% include excol.html type="end" %}

## Więcej informacji {#more-info}

> W3C WAI zapewnia szeroką gamę materiałów dotyczących różnych aspektów dostępności: [standardów](/standards-guidelines/), [edukacji](/teach-advocate/), [testowania/oceny](/test-evaluate/), [zarządzania projektami i&nbsp;polityk](/planning/). Zachęcamy do zapoznania się z&nbsp;tą witryną lub przejrzenia listy [materiałów WAI](/resources/).

Reviewer 2: ```suggestion
W3C WAI zapewnia bogaty wybór materiałów dotyczących różnych aspektów dostępności: [standardów](/standards-guidelines/), [edukacji](/teach-advocate/), [testowania/oceny](/test-evaluate/), [zarządzania projektami i&nbsp;polityk](/planning/). Zachęcamy do zapoznania się z&nbsp;naszą witryną lub przejrzenia listy [materiałów WAI](/resources/).
```
